cr.c

```diff
@@ -77,6 +78,12 @@ int dill_no_blocking(int val) {
     return old;
 }
 
+static void dill_ctx_timer_handler(int sig, siginfo_t *si, void *uc) {
+    struct dill_ctx_cr *ctx = si->si_value.sival_ptr;
+    int or = timer_getoverrun(ctx->timer);
+    if(or) ctx->do_poll = 1;
```

Comment:
I don't think so because the signal handler wouldn't be called otherwise. http://man7.org/linux/man-pages/man2/timer_getoverrun.2.html suggests that `timer_getoverrun` itself can return 0, however... so I'm being a bit pedantic in that instance.